crates/ruma-common/src/push.rs

```diff
@@ -476,6 +476,7 @@ impl ConditionalPushRule {
     ///
     /// * `event` - The flattened JSON representation of a room message event.
     /// * `context` - The context of the room at the time of the event.
+    #[allow(deprecated)]
```

Comment:
Could these be moved to the exact block/statement that the deprecated items are referenced?